app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -40,73 +40,89 @@ quote:
+If you’re not sure if you want to teach primary or secondary, you can [find out more about teaching different age groups](/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach).
 
-## Should my personal statement be different if I’m training to teach primary or secondary? 
+## Get help with your personal statement
 
-You should use your personal statement to explain why you feel passionate about teaching a specific age range or subject. 
+Writing a personal statement can feel daunting, but our teacher training advisers can help. 
 
-If you’re applying for a primary course with a subject specialism, or you’re particularly interested in certain primary subjects, you can talk about that, too. 
+Advisers can support you with:
 
-If you’re not sure if you want to teach primary or secondary, you can [find out more about teaching different age groups](/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach). 
+* what to include in your statement
 
-## Do I need school experience? 
+* how to structure your statement so you can write it in managable sections
 
-You do not need school experience to apply for teacher training, but it can help strengthen your personal statement. 
+* developing your own voice in your statement  
 
-Teacher training providers like to see that you have a good understanding of teaching, how the school system works and what your transferable skills are. You need more than just good subject knowledge and school experience can be a great way to get this. 
+$q-Josie$ 
 
-Getting some school experience can also be a good way to make sure teaching is right for you before you apply for a course. 
+You can get help with your personal statement from our [teacher training advisers](/teacher-training-advisers). They have years of teaching experience and can give you free, one-to-one support by phone, text, or email.
 
-[Find out how you could get school experience](/train-to-be-a-teacher/get-school-experience). 
+Advisers can also help you understand more about what teaching is really like, which can help improve your application.
 
-## Get help with your personal statement 
+## Get school experience
 
-You can get help with your personal statement from our [teacher training advisers](/teacher-training-advisers). They have years of teaching experience and can give you free, one-to-one support by phone, text, or email. 
+You do not need school experience to apply for teacher training, but it can help strengthen your personal statement.
 
-Advisers can also help you understand more about what teaching is really like, which can help improve your application. 
+Getting some school experience can also be a good way to make sure teaching is right for you before you apply for a course.
 
-$q-Felix$
+ [Find out how you could get school experience](/train-to-be-a-teacher/get-school-experience).
+ 
+$q-Felix$ 
```

Comment:
Does this need moving up to above the GSE h2 ie at the end of the adviser section?